Public/Src/FrontEnd/MsBuild/MsBuildWorkspaceResolver.cs

```diff
@@ -277,11 +277,11 @@ private BuildParameters.IBuildParameters RetrieveBuildParameters()
             return buildParameters;
         }
 
-        private bool TryRetrieveParsingEntryPoint(out AbsolutePath parsingEntryPoint)
+        private bool TryRetrieveParsingEntryPoint(out IEnumerable<AbsolutePath> parsingEntryPoints)

```

Comment:
>IEnumerable<AbsolutePath> [](start = 54, length = 25)

Why are you changing this to IEnumerable if you are still validating if Count ==1 (line 299) and returning an IEnumerable with a single item?

Reply:
Because of line 284. If the user specified entry points within the resolver settings, then we certainly want to use those. Otherwise, we are finding projects ourselves; whether we want to build everything or fail by default is a reasonable discussion.

I kept the previous behavior because it seems better to build less unless asked to.